packages/web-pkg/src/constants.ts

```diff
@@ -0,0 +1,78 @@
+export abstract class DavPermission {
+  static readonly Shared: string = 'S'
+  static readonly Shareable: string = 'R'
+  static readonly Mounted: string = 'M'
+  static readonly Deletable: string = 'D'
+  static readonly Renameable: string = 'R'
```

Comment:
```suggestion
  static readonly Renameable: string = 'N'
```

Needs to be 'N' according to changes above

Reply:
🤯 good catch, thank you! 🤩